### DISCUSSION TITLE
尝试发一个视频到绳网

### DISCUSSION THREAD

**Discussion by HKLHaoBin** [share121/inter-knot#2689] [其他]
https://github.com/user-attachments/assets/897bebec-2cfe-4b70-83a9-54f012991abf

**Comment by HKLHaoBin** [comment]
### 成功了!!!
但是需要将视频大小控制在10兆以内
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by Foxxxb** [comment]
其实你可以发链接的

**Comment by LShanShi** [comment]
哇哦有点酷

**Comment by youyiMC** [comment]
是否可以尝试一下发送b站嵌入代码<img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/62158b26331a045bbaabf48d1fc5c8eb_980401343708652819.png" alt="阿姬灵感" width="75" >

**Reply by HKLHaoBin** [reply]
好主意好主意 我去研究一下！

**Reply by HKLHaoBin** [reply]
并不行。。。
![Clip_2024-08-19_13-06-34](https://github.com/user-attachments/assets/e458efd3-f88c-4c34-a9f9-c296177a2acc)

**Reply by rong-xiaoli** [reply]
估计作者没有写iframe？我是萌新，不知道这是什么问题
